qmetaobject/src/qtdeclarative.rs

```diff
@@ -877,7 +883,12 @@ impl QJSValue {
         ] -> *mut c_void as "QObject *" {
             QObject *obj = self->toQObject();
             // FIXME: inheritance?
-            return obj && obj->metaObject()->inherits(mo) ? obj : nullptr;
+#if QT_VERSION >= QT_VERSION_CHECK(5, 7, 0)
+        bool inherits = obj->metaObject()->inherits(mo);
+#else
+        bool inherits = true;
```

Comment:
Here however, we really need to get the inheritance check, otherwise a variant containing a wrong QObject would cause trouble.